apps/dashboard/components/hackerform/hackform-end/hackform-end.tsx

```diff
@@ -24,6 +28,23 @@ export const HackformEnding = ({ formMetadata }: HackformEndingProps) => {
     setTimeout(() => {
       hackformUtils.reset();
     }, TIME_BEFORE_FORMSTATE_RESET);
+
+    const supabase = container.resolve(HibiscusSupabaseClient);
+    const client = supabase.getClient();
+    const accessToken = getCookie(
+      getEnv().Hibiscus.Cookies.accessTokenName
+    ).toString();
+    const refreshToken = getCookie(
+      getEnv().Hibiscus.Cookies.refreshTokenName
+    ).toString();
+    const userProfile = await supabase.getUserProfile(
+      accessToken,
+      refreshToken
+    );
+    await client
+      .from('user_profiles')
+      .update({ application_status: 3 })
+      .eq('user_id', userProfile.user_id);
```

Comment:
you can just use the `user` object from the `useHibiscusUser()` hook (line 22) instead of fetching it again